### PR TITLE
Update IPv6() for PDNAddressAllocation

### PR DESCRIPTION
**Pull request by ptsgr**
If request contains PDNType ipv46(this means that PAA contains two addresses at the same time), there is a need to check the IP addresses. By default IP() return ipv4 address. IPv6() was updated and now return ipv6 address or err if address not present.

### CODE REVIEW

**Review by wmnsk — COMMENTED** [review]
Great! Could you add a test case for that?

**Comment by wmnsk** [comment]
Thanks. I don't think it's great idea to have if clause for each case. How do you think about this instead?

```go
package ie_test

import (
	"io"
	"net"
	"testing"

	"github.com/google/go-cmp/cmp"
	"github.com/wmnsk/go-gtp/gtpv2"
	"github.com/wmnsk/go-gtp/gtpv2/ie"
)

func TestPDNAddressAllocationIP(t *testing.T) {
	cases := []struct {
		description string
		paa         *ie.IE
		pdnType     uint8
		ipv4        net.IP
		ipv6        net.IP
	}{
		{
			"PDNType IPv4",
			ie.NewPDNAddressAllocation("1.2.3.4"),
			gtpv2.PDNTypeIPv4,
			net.ParseIP("1.2.3.4"),
			nil,
		},
		{
			"PDNType IPv6",
			ie.NewPDNAddressAllocation("::1"),
			gtpv2.PDNTypeIPv6,
			nil,
			net.ParseIP("::1"),
		},
		{
			"PDNType IPv4v6",
			ie.NewPDNAddressAllocationDual("1.2.3.4", "::1", 64),
			gtpv2.PDNTypeIPv4v6,
			net.ParseIP("1.2.3.4"),
			net.ParseIP("::1"),
		},
		{
			"PDNType NonIP",
			ie.NewPDNAddressAllocation(""),
			gtpv2.PDNTypeNonIP,
			nil,
			nil,
		},
	}

	for _, c := range cases {
		t.Run(c.description, func(t *testing.T) {
			pdnType := c.paa.MustPDNType()
			if diff := cmp.Diff(pdnType, c.pdnType); diff != "" {
				t.Error(diff)
			}

			ipv4, _ := c.paa.IPv4()
			if diff := cmp.Diff(ipv4, c.ipv4); diff != "" {
				t.Error(diff)
			}

			ipv6, _ := c.paa.IPv6()
			if diff := cmp.Diff(ipv6, c.ipv6); diff != "" {
				t.Error(diff)
			}

			ip, err := c.paa.IP()
			if err == nil {
				v := ipv4
				if pdnType == gtpv2.PDNTypeIPv6 {
					v = ipv6
				}
				if diff := cmp.Diff(ip, v); diff != "" {
					t.Error(diff)
				}
			} else if err != io.ErrUnexpectedEOF {
				t.Error(err)
			}
		})
	}
}
```

**Comment by ptsgr** [comment]
@wmnsk Done, pls check

File: gtpv1/tunnel_linux.go [hunk 1]
```diff
@@ -24,18 +24,18 @@ const (
 // EnableKernelGTP enables Linux Kernel GTP-U.
 // Note that this removes all the existing userland tunnels, and cannot be disabled while
 // the program is working (at least at this moment).
-//
```
Comment (wmnsk) [inline gtpv1/tunnel_linux.go:27]: Please don’t remove this so that godoc parser can regard the whole comment as a documentation.

**Review by wmnsk — APPROVED** [review]
Thanks! One more trivial thing but LGTM overall.

File: gtpv1/tunnel_linux.go (hunk 1 above)
Comment (ptsgr) [reply gtpv1/tunnel_linux.go:27]: Screen Shot 2022-09-01 at 4 12 51 PM

Tried to fix a linter error related to syntax complexity

**Comment by ptsgr** [comment]
LGTM for me too! Thank you!

File: gtpv1/tunnel_linux.go (hunk 1 above)
Comment (wmnsk) [reply gtpv1/tunnel_linux.go]: OK, noted as an issue 👍

**Pull request merged**